lib/repl.js

```diff
@@ -298,6 +298,7 @@ function REPLServer(prompt,
     configurable: true
   });
 
+  this.allowBlockingCompletions = !!options.allowBlockingCompletions;
```

Comment:
I’m not publicly documenting this because this is a) supposed to be temporary and b) that would make it semver-minor, and this should be backported to v14.x without the extensive process that the LTS team has for semver-minor commits.